src/docs/Doxyfile

```diff
@@ -1828,7 +1828,7 @@ COMPACT_LATEX          = NO
 # The default value is: a4.
 # This tag requires that the tag GENERATE_LATEX is set to YES.
 
-PAPER_TYPE             = a4wide
+PAPER_TYPE             = a4
```

Comment:
Yeah, apparently a4wide doesn't exist any more. I don't think anything is actually generating paper docs, so it probably doesn't matter in the least what it's set to. (It's been a4wide since the file was first created ages ago, and the history suggests it may just have been a default then.)